common/infallible/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-mod math;
+pub mod math;
```

Comment:
You could export the type you want as is done below instead of making the module public.